### DISCUSSION TITLE
Invisor Icon...

### DISCUSSION THREAD

**Discussion by OMGzilla-git** [elrumo/macOS_Big_Sur_icons_replacements#1455] [Icon request] [answered]
I'd love to get a Big Sur-style icon for Invisor that looks great, but this is going to take some mad skills to pull off. There's an existing one someone made, but has a simple red background for the original icon. I'm hoping someone can bust out something that looks nicer. Here's the original:

<img width="512" alt="Invisor" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/85633412/f8c856b7-ee34-48c6-91e9-7254850a4bb6">

One challenge I want to point out to doing anything using the original icon is that it's slightly translucent. My personal thought for an icon was to just take the foreground strip of the icon and adjust it so that it fills to the rounded corners of the Big Sur-style icon boundaries. Here's a crude example where I simply cropped it in:

![Temp 2](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/85633412/c688366a-b744-437a-a719-c6e8fbeceb11)

That's obviously only "half" an icon, but illustrates what I'm shooting for. One problem with this is that you can see remnants of the second film strip that was part of the original icon showing through due to the translucency, and that's no good.

So! Any aces up for a challenge?

**Comment by Dreamkiller711** [comment]
Just opened the forum for the first time to look for Invisor.

I second that :)

**Comment by Karukhai** [comment, marked as answer]
Hi, I recreated the Invisor logo in this style. You can find it by searching or inside my profile: https://macosicons.com/#/u/kenji

**Reply by OMGzilla-git** [reply]
Very nicely done! And compared to a real film strip, your border with the holes for a real-life projector aligns to the "long" side of a widescreen frame in a film strip, unlike the original icon. Much appreciated!

**Reply by Karukhai** [reply]
I’m glad you like it! Thank you very much!
